eogrow/cli.py

```diff
@@ -99,11 +101,9 @@ def run_pipeline_on_cluster(
     if start_cluster:
         start_cluster_if_needed(cluster_yaml)
 
-    raw_configs = [interpret_config_from_dict(config) for config in collect_configs_from_path(config_path)]
     remote_path = generate_cluster_config_path(config_path)
-
     with NamedTemporaryFile(mode="w", delete=True, suffix=".json") as local_path:
-        json.dump(raw_configs, local_path)
+        json.dump(collect_configs_from_path(config_path), local_path)
```

Comment:
this now dumps the crude configs, right? I guess that's more in line with the configs we have on disk anyway, but is there a specific reason for the change?

Reply:
previously the variables were not instantiated correctly. If you did `eogrow-ray -v "x:1"` it wouldn't apply the variable, because it interpreted the config without the CLI variables

The `eogrow-ray` command is just used to execute the appropriate `eogrow` command on the cluster, and that command expects a crude config (since **it** does the instantiation).